sdk/batch/Azure.Compute.Batch/samples/Sample1_CreatePool_Job_Task.md

```diff
@@ -0,0 +1,241 @@
+
+## Authenticating the Azure.ResourceManager `ArmClient`
+
+In order to create Batch Pool from the Azure.Managment.Batch library you will need to instantiate an Armclient. To create an authenticated client and start interacting with Microsoft Azure resources, see the [quickstart guide here](https://github.com/Azure/azure-sdk-for-net/blob/main/doc/dev/mgmt_quickstart.md).
+
+```C#
+var credential = new DefaultAzureCredential();
+ArmClient _armClient = new ArmClient(credential);
+```
+
+
+### Pool Creation
+
+Batch operations in the **Azure.Management.Batch** sdk are preformed from a BatchAccountResource object, to get a BatchAccountResource object you can query the armclient for the resource id of your Batch account.
+```C#
+BatchAccountResource batchAccount = await _armClient.GetBatchAccountResource("your-batch-account-resource-id").GetAsync();
+```
+
+With the BatchAccountResource you can create a pool with the [batchAccount.GetBatchAccountPools().CreateOrUpdateAsync](https://learn.microsoft.com/dotnet/api/azure.resourcemanager.batch.batchaccountpoolcollection.createorupdateasync?view=azure-dotnet) command
+```C#
+
+var poolName = "HelloWorldPool";
+var imageReference = new BatchImageReference()
+{
+    Publisher = "canonical",
+    Offer = "0001-com-ubuntu-server-jammy",
+    Sku = "22_04-lts",
+    Version = "latest"
+};
+string nodeAgentSku = "batch.node.ubuntu 22.04";
+
+BatchAccountPoolResource pool = (await batchAccount.GetBatchAccountPools().CreateOrUpdateAsync(WaitUntil.Completed, poolName, new BatchAccountPoolData()
+{
+    VmSize = "Standard_DS1_v2",
+    DeploymentConfiguration = new BatchDeploymentConfiguration()
+    {
+        VmConfiguration = new BatchVmConfiguration(imageReference, nodeAgentSku)
+    },
+    ScaleSettings = new BatchAccountPoolScaleSettings()
+    {
+        FixedScale = new BatchAccountFixedScaleSettings()
+        {
+            TargetDedicatedNodes = 1
+        }
+    }
+})).Value;
+```
+
+## Authenticating the Azure.Compute.Batch `BatchClient`
+Creation of Batch jobs and tasks can only be preformed with the `Azure.Compute.Batch` sdk.  A `BatchClient` object is needed to preform Batch operations and can be created using [Microsoft Entra ID authtentication](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/identity/Azure.Identity/README.md) and the Batch account endpoint  
+
+```C#
+var credential = new DefaultAzureCredential();
+BatchClient _batchClient = new BatchClient(
+    new Uri("https://examplebatchaccount.eastus.batch.azure.com"),
+    credential);
+```
+
+### Job Creation
```

Comment:
```suggestion
### Job creation
```

nit:  The Microsoft style guide requires that we use sentence-style capitalization in titles and headings.   _([source](https://learn.microsoft.com/style-guide/capitalization))_